packages/tasks/src/model-libraries.ts

```diff
@@ -114,6 +114,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"npz"`,
 	},
+	birder: {
+		prettyLabel: "Birder",
+		repoName: "Birder",
+		repoUrl: "https://gitlab.com/birder/birder",
+		filter: false,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
Out of curiousity, wouldn't the `config.json` not work as well? I see that the downloads already show up for: https://huggingface.co/birder-project/davit_tiny_il-all

Reply:
In the Birder library, there are a few ways to load a model (https://gitlab.com/birder/birder/-/blob/main/docs/internal/api.md?ref_type=heads)

The load with config is not a popular one, usually only the weights are being accessed.

Reply:
did you see by the way @sashavor? https://gitlab.com/birder/birder